### DISCUSSION TITLE
昨天一夜之间无法获取IPV6网卡信息，正常运行被出错，涉及三台电脑

### DISCUSSION THREAD

**Discussion by HeroGotoWork** [jeessy2/ddns-go#1059] [General]
每天都是正常运行的，没有做过任何的更新，更改，没有关过电脑。突然在运行正常的一夜之间三台电脑全部无法获取网卡信息
![微信图片_20240326161809](https://github.com/jeessy2/ddns-go/assets/165002848/d3931078-a8ba-4d5f-87ec-f61a1d6d8cd8)

**Comment by jeessy2** [comment]
那就没有IPv6了吧。进入电脑查看，是否存在

**Reply by HeroGotoWork** [reply]
![微信截图_20240327133247](https://github.com/jeessy2/ddns-go/assets/165002848/56c0d0f2-98ac-4c89-806d-e6cc56889ff3)
存在应该，没有动过任何硬件软件。

**Reply by jeessy2** [reply]
截图的IPv6是DNS服务器，IPv6地址没看到公网的

**Reply by jeessy2** [reply]
测下IPv6

**Reply by HeroGotoWork** [reply]
果然
没有检测到 IPv6 地址

这真是奇怪了，运营商一夜之间把IPV6关了？

**Reply by HeroGotoWork** [reply]
老板， IPV4动态更新是不是无法连接？

**Reply by jeessy2** [reply]
IPV4要真公网IP

**Reply by HeroGotoWork** [reply]
我打电话给电信了，我要反馈这个问题。都用了几年了，说关就关。
